public/app/features/serviceaccounts/ServiceAccountRoleRow.tsx

```diff
@@ -37,7 +37,7 @@ export class ServiceAccountRoleRow extends PureComponent<Props> {
               userId={serviceAccount.id}
               orgId={serviceAccount.orgId}
               builtInRole={serviceAccount.role}
-              onBuiltinRoleChange={(newRole) => onRoleChange(newRole, serviceAccount)}
+              onBuiltinRoleChange={(newRole) => onRoleChange(newRole)}
```

Comment:
```suggestion
              onBuiltinRoleChange={onRoleChange}
```
No need to create an extra function fi the arguments are the same.